pkg/apis/m3dboperator/v1/cluster.go

```diff
@@ -63,9 +63,12 @@ type ClusterSpec struct {
 	// NumberOfShards defines how many shards in total
 	NumberOfShards int32 `json:"numberOfShards"`
 
-	// Zones specifies a map of key-value pairs. Defines which zones
+	// NumberOfPods defines how many PODs should run in total
```

Comment:
I'm having trouble figuring out if ClusterSpec is the desired state or the current state?

Reply:
Desired state (goal state) for the cluster to converge on, see previous comment.